PubNubUser/Sources/User+PubNub.swift

```diff
@@ -205,7 +205,7 @@ public extension PubNubUserInterface {
   ) {
     let router = ObjectsUUIDRouter(
       .all(
-        customFields: includeCustom,
+        include: PubNub.IncludeFields(custom: includeCustom).userIncludeFields,
```

Comment:
Will it make more sense to have something like this:
```suggestion
        include: PubNub.UserIncludeFields(custom: includeCustom),
```

Reply:
I agree, it makes sense. I'll introduce it and let you know when this becomes available